<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Camera</class>
 <widget class="QMainWindow" name="Camera">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #F5EFEE;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>500</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="bottomMargin">
     <number>15</number>
    </property>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">
QLabel {
	color: rgb(50, 50, 100);
    font-weight: bold;
    font-size: 12px; /* Adjust the font size as needed */
 }</string>
      </property>
      <property name="text">
       <string>Sign Language to Speech Translator</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1">
       <property name="spacing">
        <number>5</number>
       </property>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="palette">
          <palette>
           <active>
            <colorrole role="Button">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>245</red>
               <green>239</green>
               <blue>238</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>245</red>
               <green>239</green>
               <blue>238</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Window">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>245</red>
               <green>239</green>
               <blue>238</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="Button">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>245</red>
               <green>239</green>
               <blue>238</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>245</red>
               <green>239</green>
               <blue>238</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Window">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>245</red>
               <green>239</green>
               <blue>238</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="Button">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>245</red>
               <green>239</green>
               <blue>238</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>245</red>
               <green>239</green>
               <blue>238</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Window">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>245</red>
               <green>239</green>
               <blue>238</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="previewPage">
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="viewfinderPage">
          <property name="styleSheet">
           <string notr="true">background-color: #E8E3E1</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QVideoWidget" name="viewfinder" native="true"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="styleSheet">
          <string notr="true">background-color: #E8E3E1;
border-style: none;
color: black;

</string>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="translationTab">
          <property name="styleSheet">
           <string notr="true">background-color: #E8E3E1</string>
          </property>
          <attribute name="title">
           <string>Trans</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_2" stretch="8,2">
           <item>
            <widget class="QWidget" name="topArea" native="true">
             <widget class="QTextEdit" name="translationDisplay">
              <property name="geometry">
               <rect>
                <x>3</x>
                <y>0</y>
                <width>171</width>
                <height>321</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: #D9D1CE</string>
              </property>
              <property name="lineWidth">
               <number>3</number>
              </property>
              <property name="sizeAdjustPolicy">
               <enum>QAbstractScrollArea::AdjustIgnored</enum>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="bottomArea" native="true">
             <layout class="QVBoxLayout" name="verticalLayout_3" stretch="6,1">
              <item>
               <widget class="QTextEdit" name="translateInput">
                <property name="styleSheet">
                 <string notr="true">background-color: #F5F1F0</string>
                </property>
                <property name="placeholderText">
                 <string>Text to Translate ...</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="translateButton">
                <property name="styleSheet">
                 <string notr="true">background-color: #D9D1CE;
color: black;</string>
                </property>
                <property name="text">
                 <string>Translate</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="dictionaryTab">
          <property name="styleSheet">
           <string notr="true">background-color: #E8E3E1</string>
          </property>
          <attribute name="title">
           <string>Dict</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,8">
           <item>
            <widget class="QWidget" name="searchArea" native="true">
             <layout class="QVBoxLayout" name="verticalLayout_5" stretch="3,1">
              <item>
               <widget class="QTextEdit" name="dictionaryInput"/>
              </item>
              <item>
               <widget class="QPushButton" name="dictionaryButton">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>Coming Soon</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="dictDisplayArea" native="true">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_6" stretch="1">
              <item>
               <widget class="QTextEdit" name="dictionaryTextOutput">
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="historyTab">
          <attribute name="title">
           <string>Hist</string>
          </attribute>
          <widget class="QTextEdit" name="historyDisplay">
           <property name="geometry">
            <rect>
             <x>3</x>
             <y>3</y>
             <width>181</width>
             <height>421</height>
            </rect>
           </property>
           <property name="documentTitle">
            <string>Coming Soon</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string>Coming Soon</string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionStartCamera">
   <property name="text">
    <string>StartCamera</string>
   </property>
  </action>
  <action name="actionStopCamera">
   <property name="text">
    <string>StopCamera</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header>qvideowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
